hassio/src/dashboard/hassio-dashboard.ts

```diff
@@ -33,6 +34,11 @@ class HassioDashboard extends LitElement {
 
   @property({ attribute: false }) public hassOsInfo!: HassioHassOSInfo;
 
+  protected firstUpdated(changedProps) {
+    super.firstUpdated(changedProps);
+    this._postUpdateDialog();
```

Comment:
Is there a good event that can be listened on here?
Currently, if the user now refreshes while doing the upgrade this will trigger, I guess that is fine.

I tried listening to the connection -> ready event, but that fires too early, the dialogs would not show.

Reply:
It will also trigger when I navigated away during the download/update and come back a week later, that is weird I think?

Reply:
If we want to go this way, we should store a time or something to make sure it only shows right after the update

Reply:
And if I updated in the meantime to a version higher it will say my update failed

Reply:
All true, which is why I asked about a event

Reply:
The event will not fix it if the user is not at this page when it is fired?

Reply:
No matter how we do this, it will be some weird situations.
I'll take this part out of the PR, and we can implement something (if needed) when something for <https://github.com/home-assistant/supervisor/issues/1946> is in place.